src/Components/SingleArticle.jsx

```diff
@@ -29,20 +32,59 @@ export default function SingleArticle() {
       ]) => {
         setArticleInfo([article]);
         setCommentsList(comments);
-        setIsLoading(false)
+        setIsLoading(false);
       }
     );
   }, []);
 
-  return ( isLoading ? <h1>Loading Article</h1> : 
+  function handleSubmit(e) {
+    e.preventDefault();
+    const body = {
+      username: signedInUser.username,
+      body: commentInput,
+    };
+    axios
+      .post(
+        `https://northcoders-news-03ck.onrender.com/api/articles/${articleId}/comments`,
+        body
+      )
```

Comment:
Ideally we'd have our axios requests in functions in a separate utils file like api.js, then require them in to use them in our components.